tests/test_wave_01.py

```diff
@@ -118,13 +118,12 @@ def test_moves_movie_from_watchlist_to_empty_watched():
     # Assert
     assert len(updated_data["watchlist"]) == 0
     assert len(updated_data["watched"]) == 1
+    assert updated_data["watched"][0]["title"] == MOVIE_TITLE_1
+    assert updated_data["watched"][0]["genre"] == GENRE_1
+    assert updated_data["watched"][0]["rating"] == RATING_1
+    assert updated_data["watchlist"] == []
```

Comment:
👍  Nice work! I would remove the last test as the first test of checking if the watchlist is empty is sufficient enough.